session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -589,6 +589,47 @@ public SessionDataSet executeRawDataQuery(List<String> paths, long startTime, lo
     }
   }
 
+  /**
+   * only: select last status from root.ln.wf01.wt01 where time >= 1621326244168; <PrefixPath> +
+   * <suffixPath> = <TimeSeries>
+   *
+   * @param suffixPath root.ln.wf01.wt01
+   * @param prefixPath status
```

Comment:
Change the comment to a reasonable one, we should first explain the param's meaning and then we can give some examples.

Reply:
ok ，Has been modified and submitted